cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -229,14 +229,7 @@ handlePeersList tr nodeKernIORef lvbe = forever $ do
   peers <- getCurrentPeers nodeKernIORef
   storePeersInLiveView peers lvbe
   tracePeers tr peers
-  -- We couldn't get these values during initialisation, so pass them now.
-  (capacity, capacityBytes) <- getMempoolData
-  storeMempoolDataInLiveView capacity capacityBytes lvbe
   threadDelay 2000000 -- 2 seconds.
- where
-  getMempoolData = do
-    nkd <- readIORef nodeKernIORef
-    return (nkdMempoolCapacity nkd, nkdMempoolCapacityBytes nkd)
```

Comment:
if you don't access them anymore, then they do not need to be prepared and stored in that data structure.

Reply:
Agree. Will fix now.

Reply:
Done.